### PR TITLE
Git release

### PR DESCRIPTION
**Pull request by ahmadiq**

### CODE REVIEW

File: vars/mavenCanaryRelease.groovy [hunk 1]
```diff
@@ -52,7 +52,14 @@ def call(body) {
         bayesianScanner(body)
     }
 
-    sonarQubeScanner(body)
+    //use sonarqube waitforqualitygate stage in project jenkinsfile instead
+    //sonarQubeScanner(body)
```
Comment (rasheedamir) [inline vars/mavenCanaryRelease.groovy]: we shouldn't make a release if quality gate fails!

Comment (ahmadiq) [inline vars/mavenCanaryRelease.groovy]: ok that change will go in the project jenkinsfile. I'll revisit the jenkinsfile in MovieManager, and make appropriate change as a demo.

File: vars/mavenCanaryRelease.groovy [hunk 2]
```diff
@@ -52,7 +52,14 @@ def call(body) {
...
+
+    //push release git branch and tag
+    gitRelease {
```
[the remark this answers is not in the document]
Comment (rasheedamir) [reply vars/mavenCanaryRelease.groovy]: @ahmadiq is this taken care of?

[the remark this answers is not in the document]
Comment (ahmadiq) [reply vars/mavenCanaryRelease.groovy]: yes it's part of gitRelease, but i've moved it out of mavenCanaryRelease, and gitRelease will be called from the project Jenkinsfile

**Pull request merged**